src/api/app/assets/javascripts/webui2/staging_workflow.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
+    
+  $(".delete-spinner").hide();
```

Comment:
please don't. this is executed on every page. Put this into a function and then call the function from content_for :ready_function only on the template you have a delete-spinner on

Reply:
done :)